File: python/1 - average.py
how_many = int(input("How many numbers do you want to type in? "))

sum = 0

for n in range(how_many):
    number = int(input(f"Number #{n+1}: "))
    sum += number


average = sum / how_many
print(f"Your average is: {average:.2f}")

File: Go/1 - average.go
package main

import "fmt"

func average() float64 {

	var how_many int
	var sum float64

	fmt.Print("How many numbers do you want to type in? ")
	fmt.Scanln(&how_many)

	for i := 0; i < how_many; i++ {
		temp := 0.0

		fmt.Printf("Number #%d ", i+1)
		fmt.Scanln(&temp)
		sum += temp
	}

	return sum / float64(how_many)
}
